components/create/Project.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import CreatePortfolioContext from 'components/create/CreatePorfolioContext';
+import { TextField, Stack, Button } from '@mui/material';
+import { useContext, useRef, RefObject } from 'react';
+import { IProject, ILink } from 'models/data/';
+import { StyledTextField, StyledButton } from 'components/create/Styled';
+
+const Project = () => {
+  const formRef = useRef() as RefObject<HTMLFormElement>;
+  const entry: IProject = {
+    title: '',
+    picture: '',
+    summary: '',
+    links: [],
+  };
+  const source: ILink = {
+    label: '',
+    url: '',
+  };
+  const demo: ILink = {
+    label: '',
+    url: '',
+  };
+  const { projects, setProjects } = useContext(CreatePortfolioContext);
+  const onSubmit = async () => {
+    entry.links?.push(source);
+    entry.links?.push(demo);
```

Comment:
What if I don't have any links? Maybe you can check if the URL is blank or not

Reply:
fixed

Reply:
Fixed with the LinksEditor.